src/main/java/echoServer/BufferedReaderWrapper.java

```diff
@@ -1,20 +1,33 @@
 package echoServer;
 import java.io.BufferedReader;
 import java.io.IOException;
-import java.io.InputStreamReader;
-import java.net.Socket;
 
 public class BufferedReaderWrapper implements ClientReadable {
 
-
     public BufferedReader bufferedReader;
 
     public BufferedReaderWrapper(BufferedReader bufferedReader) throws IOException {
         this.bufferedReader = bufferedReader;
     }
 
+
     @Override
-    public String readLine() throws IOException {
-        return bufferedReader.readLine();
+    public String readAllLines() throws IOException {
+        String CRLF = "\r\n";
+        String line;
+        StringBuilder sb = new StringBuilder();
```

Comment:
Also have been quite confused as to how this compiles without your importing the `StringBuilder` class. Not sure if it's because you import `java.io.*` in `EchoServer` (which by the way, you ought to not allow wildcard imports).

Reply:
I got rid of the `java.io.*` in `EchoServer` but it still compiled... how strange